postgres-appliance/Dockerfile

```diff
@@ -64,31 +122,13 @@ RUN export DEBIAN_FRONTEND=noninteractive \
             python3 python3-pkg-resources python3-setuptools \
             ${BUILD_PACKAGES} \
 
-    # install pam_oauth2.so
-    && curl -s -L https://github.com/zalando-incubator/pam-oauth2/archive/$PAM_OAUTH_COMMIT.zip > pam-oauth2.zip \
-    && unzip pam-oauth2.zip \
-    && cd pam-oauth2-$PAM_OAUTH_COMMIT \
-    && curl -s -L https://github.com/zserge/jsmn/archive/$JSMN_COMMIT.zip > jsmn.zip \
-    && unzip jsmn.zip \
-    && rm -fr jsmn && mv jsmn-$JSMN_COMMIT jsmn \
-    && make install \
-    && cd .. \
-    && rm -fr pam-oauth2* \
-
-    # install extensions for old postgres versions
-    && export OLD_PATH=$PATH \
-    && for version in ${PGOLDVERSIONS}; do \
-        apt-get install -y postgresql-server-dev-${version} \
-        && export PATH=/usr/lib/postgresql/${version}/bin:$OLD_PATH \
-        && for extension in ${PGXN_EXTENSIONS}; do pgxn install $extension; done \
-        && apt-get purge -y postgresql-server-dev-${version}; \
-    done \
-    && export PATH=$OLD_PATH \
-
     && pip3 install pip --upgrade \
     && pip3 install --upgrade packaging appdirs requests pystache patroni==$PATRONIVERSION \
             gcloud boto wal-e==$WALE_VERSION \
-    && for extension in ${PGXN_EXTENSIONS}; do pgxn install $extension; done \
+
+    # https://github.com/wal-e/wal-e/issues/318
+    && sed -i 's/^\(    for i in range(0,\) num_retries):.*/\1 100):/g' /usr/local/lib/python3.4/dist-packages/boto/utils.py \
```

Comment:
The page you've linked to says it's only used for AWS CLI. The boto (2) page explicitly lists configuration files boto searches: http://docs.pythonboto.org/en/latest/boto_config_tut.html